the-events-calendar/tribe-common
# Notice code updated to adhere to more modern Standards

QA Instructions:
- Install The Events Calendar or Event Tickets with the latest version
- Ensure the dismissal of 3 or 4 different Notices are working as expected.
- Check the `wp_usermeta` that you see `tribe-dismiss-notice` for your User ID show up as you click.

📹 https://i.bordoni.me/SpX88bbc
